doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_fota_update.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
```

Comment:
I wonder if we could e.g. link the page under https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_dfu.html or add the content as an example on https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_smp.html (to avoid orphan page)

Reply:
This is up to the Charon and the Charon documentation teams to decide. My intention is to create linkable steps for SUIT-based applications, not a full-blown documentation page.

Reply:
@tomchy recommended to set up this page as an orphan page and that's why I did it like that (probably due to the upcoming documentation restructure)

Reply:
@kapi-no are these steps only supposed to be "quoted" from other doc pages? If this is the only scope of this PR, this should be done through a txt document located in doc\nrf\includes